lib/twine/formatters.rb

```diff
@@ -1,30 +1,13 @@
-require 'twine/formatters/abstract'
-require 'twine/formatters/android'
-require 'twine/formatters/apple'
-require 'twine/formatters/flash'
-require 'twine/formatters/gettext'
-require 'twine/formatters/jquery'
-require 'twine/formatters/django'
-require 'twine/formatters/tizen'
-
 module Twine
   module Formatters
-    @formatters = [Formatters::Apple, Formatters::Android, Formatters::Gettext, Formatters::JQuery, Formatters::Flash, Formatters::Django, Formatters::Tizen]
+    @formatters = []
 
     class << self
       attr_reader :formatters
-
-      ###
-      # registers a new formatter
-      #
-      # formatter_class - the class of the formatter to register
-      #
-      # returns array of active formatters
-      #
-      def register_formatter formatter_class
-        raise "#{formatter_class} already registered" if @formatters.include? formatter_class
-        @formatters << formatter_class
-      end
```

Comment:
True. I'll put it back in. What's your opinion about a legacy warning informing plugin maintainers (and users) to rather use `Twine::Formatters.formatters << My::Formatters::Awesome.new`? This way of registering formatters gives (plugin) developers the freedom to write formatters requiring initializer parameters.